packages/styles/src/sass/components/text-input/_index.scss

```diff
@@ -0,0 +1,233 @@
+.cbp-input__label {
+  color: var(--cbp-color-text-darkest);
+  display: inline-block;
+  font-size: var(--cbp-font-size-body);
+  font-style: italic;
+  font-weight: var(--cbp-font-weight-bold);
+  margin-bottom: var(--cbp-space-2x);
+}
+
+.cbp-input__description {
+  color: var(--cbp-color-text-darkest);
+  font-size: var(--cbp-font-size-subhead);
+  font-style: italic;
+  font-weight: var(--cbp-font-weight-medium);
+  margin-bottom: var(--cbp-space-2x);
+
+  &--error {
+    color: var(--cbp-color-danger-base);
+  }
+}
+
+.cbp-input {
+  background-color: var(--cbp-color-white);
+  border-color: var(--cbp-color-interactive-neutral-base);
+  border-style: solid;
+  border-width: var(--cbp-border-size-md);
+  border-radius: var(--cbp-border-radius-soft);
+  color: var(--cbp-color-text-darkest);
+  font-size: var(--cbp-font-size-body);
+  font-weight: var(--cbp-font-weight-regular);
+  letter-spacing: var(--cbp-letter-spacing-regular);
+  min-height: 2.25rem;
+  padding: var(--cbp-space-2x);
+  width: 100%;
+
+  &::placeholder {
+    font-style: italic;
+  }
+
+  &:hover {
+    border-color: var(--cbp-color-interactive-neutral-darker);
+  }
+
+  &:focus {
+    border-color: var(--cbp-color-interactive-focus-dark);
+    outline: 0;
+  }
+
+  &:read-only {
+    background-color: var(--cbp-color-interactive-neutral-lighter);
+    border-color: var(--cbp-color-interactive-neutral-light);
+    cursor: not-allowed;
+
+    &:hover,
+    &:focus {
+      border-color: var(--cbp-color-interactive-neutral-light);
+    }
+  }
+
+  &:disabled {
+    background-color: var(--cbp-color-interactive-disabled-light);
+    border-color: var(--cbp-color-interactive-disabled-dark);
+    cursor: pointer;
+
+    &:hover {
+      border-color: var(--cbp-color-interactive-neutral-darker);
+    }
+
+    &:focus {
+      border-color: var(--cbp-color-interactive-focus-dark);
+    }
+  }
+
+  &:invalid {
+    background-color: var(--cbp-color-danger-lighter);
+    border-color: var(--cbp-color-danger-base);
+
+    &:focus {
+      border-color: var(--cbp-color-interactive-focus-dark);
+    }
+    
+    &:hover {
+      border-color: var(--cbp-color-danger-dark);
+    }
+
+    &:hover:focus {
+      border-color: var(--cbp-color-interactive-focus-dark);
+    }
+  }
+
+  // This removes the default increment arrows in the number input field
+  &::-webkit-outer-spin-button,
+  &::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0; /* Margins still appear even if hidden */
+  }
+}
+
+textarea.cbp-input {
+  min-height: 6.75rem;
+  padding-top: var(--cbp-space-2x);
+  padding-bottom: var(--cbp-space-2x);
+  resize: none;
+}
+
+// Inputs with <button> attached to the end
+.cbp-input-group {
+  display: flex;
+
+  & > .cbp-input {
+    border-right: 0;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    flex: 1 1 auto;
+
+    &:disabled {
+      cursor: pointer;
```

Comment:
can we delete and handle at a higher level? Doesn't need the added specificity.

Reply:
removed